homeassistant/components/wallbox/number.py

```diff
@@ -24,7 +23,7 @@ class WallboxNumberEntityDescription(NumberEntityDescription):
     CONF_MAX_CHARGING_CURRENT_KEY: WallboxNumberEntityDescription(
         key=CONF_MAX_CHARGING_CURRENT_KEY,
         name="Max. Charging Current",
-        device_class=DEVICE_CLASS_CURRENT,
+        device_class="current",
```

Comment:
What is the purpose of setting the device class here?
Would it not be better to simply set the icon?

Reply:
Well... entity numbers do not have device classes...
So; this is a bug.